### ISSUE TITLE
Keyboard operation for accessibility

### ISSUE THREAD

**Issue by pniebrzydowski** [ekwonye-richard/react-flags-select#8]
There is currently no support for keyboard navigation through the component. It should be possible to be able to open the dropdown and select a country without using the mouse, but it currently requires clicking.

**Comment by ekwonye-richard** [comment]
Thanks @pniebrzydowski for the PR, I also did some styling to the focus state which is available in the latest version.
